build.gradle.kts

```diff
@@ -141,15 +184,15 @@ allprojects {
         )
     }
 
-    dependencies {
-        errorprone(rootProject.libs.errorprone)
-        errorprone(rootProject.libs.nullaway)
-
-        testImplementation(rootProject.libs.junit.jupiter.api)
-        testImplementation(rootProject.libs.org.hamcrest.core)
-        testImplementation(rootProject.libs.org.mockito.core)
-        testImplementation(rootProject.libs.org.awaitility.awaitility)
+    // IntelliJ IDE
+    apply(plugin = "idea")
+    idea {
+        module {
+            isDownloadJavadoc = true
+            isDownloadSources = true
 
-        testRuntimeOnly(rootProject.libs.junit.jupiter.engine)
+            testSourceDirs.addAll(java.sourceSets["intTest"].java.srcDirs)
```

Comment:
note: I am unsure if these two lines have any effect. TBD in a later investigation.